integration/testnet/network.go

```diff
@@ -731,16 +771,18 @@ func (net *FlowNetwork) AddNode(t *testing.T, bootstrapDir string, nodeConf Cont
 
 			nodeContainer.bindPort(hostPort, containerPort)
 
-			hostMetricsPort := testingdock.RandomPort(t)
-			containerMetricsPort := "8080/tcp"
+			// hostMetricsPort := testingdock.RandomPort(t)
+			// containerMetricsPort := "8080/tcp"
 
-			nodeContainer.bindPort(hostMetricsPort, containerMetricsPort)
-			nodeContainer.Ports[ColNodeMetricsPort] = hostMetricsPort
-			net.AccessPorts[ColNodeMetricsPort] = hostMetricsPort
-			net.MetricsPortsByContainerName[nodeContainer.Name()] = hostMetricsPort
+			// nodeContainer.bindPort(hostMetricsPort, containerMetricsPort)
+			// nodeContainer.Ports[ColNodeMetricsPort] = hostMetricsPort
+			// net.AccessPorts[ColNodeMetricsPort] = hostMetricsPort
+			// net.MetricsPortsByContainerName[nodeContainer.Name()] = hostMetricsPort
```

Comment:
is the commented-out code still needed? If so, a comment would be great indicating when, e.g. 

```golang
\\ use the following lines for testing <something>
```